### DISCUSSION TITLE
EQ7 reflections

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/library#295] [General]
Overall, this did not go well today. I felt I had to talk a lot to explain what was going on because there is very little inquiry in the section.

[Activity 1.7.2](https://library.tbil.org/2024/precalculus/EQ7.html#quadratic-inequality-coeff-greater-than) went okay, but depended on looking at a graph. Which is fine, but my students weren't super comfortable with the graphs. (And they learn to graph them later in PR1). We talked about the idea of being above, on, or below the axis. And it did help to reinforce interval notation.

But then [Definition 1.7.3](https://library.tbil.org/2024/precalculus/EQ7.html#def-sign-chart) tells what a sign chart is and then is never mentioned again. I'm also not sure why shading is mentioned as ambiguous in this definition. And why it is "often decorated with a + or -".  And how do we get the + or - ? Would be helpful to have an activity about making a sign chart. Especially to lead them to the factored form being easier because then you can test just for the sign instead of the actual value. 

[Activity 1.7.5](https://library.tbil.org/2024/precalculus/EQ7.html#quadratic-inequality-coeff-less-than-0) asks the students to solve an inequality algebraically for the first time, but they aren't told what that means or how to approach it. Or how to test values and make the sign chart. 

In [Activity 1.7.8(a)](https://library.tbil.org/2024/precalculus/EQ7.html#activity-rational-inequality-1-2) it would help to have them rewrite the inequality where all terms are on one side. My students could see that B or C could both potentially be helpful. But there was confusion over keeping the inequality sign the correct direction. (It "feels" better to move everything to the left. But it's okay to move things to the right as long as you know how to interpret something like  0 > some expression and that it's the same a some expression < 0.)

Is partition value a standard term? Referencing [Definition 1.7.9](https://library.tbil.org/2024/precalculus/EQ7.html#def-partition-values). I feel like I've called them boundary points. Or critical points (though I guess technically those generally deal with the derivative). Either way, maybe this should be introduced earlier to clarify sign charts.

There also needs to be some sort of explanation/motivation for why we can't solve the rational inequalities like we did with rational equations by multiplying through to clear fractions. It's not obvious why that doesn't work. Motivation for getting one rational expression.

Honestly, I just abandoned the activity book except picking out the examples so I didn't have to make them up on the spot. I mostly lectured, with some bouncing around and getting them to finish the problems.

**Comment by AbbyANoble** [comment]
Also, an activity with multiple parts where they solve something like:

- $x^2+8x+15>0$
- $x^2+8x+15<0$
- $x^2+8x+15\geq 0$
- $x^2+8x+15\leq 0$

To make the point that you can answer all of them by doing the "work" only once.

**Reply by siwelwerd** [reply]
These are my favorite, because most of them just start over and then get halfway through the second (or third) one and start realizing they are doing the same thing over (and over) again.

**Comment by tdegeorge** [comment]
I am going to attempt to fix this section.....Looks like a lot though!
